index.html

```diff
@@ -1,20 +1,15 @@
-<!DOCTYPE html> testing gibberish
+<!DOCTYPE html>
 <html>
   <head>
     <title>Doggo LLC</title>
     <meta charset="utf-8" />
   </head>
-  <body>
     <video width="100%" autoplay loop muted>
```

Comment:
all page content goes after the <body> tag. move the video code block after the body and wrap it in a <header></header> tag.